ext/AvizExt/viz/sensitivity.jl

```diff
@@ -530,8 +539,7 @@ function ADRIA.viz.outcome_map!(
     g::Union{GridLayout,GridPosition},
     rs::ResultSet,
     outcomes::NamedDimsArray,
-    factors::Vector{String};
-    opts::Dict=Dict(),
+    factors::Vector{Symbol};
     axis_opts::Dict=Dict(),
 )
```

Comment:
Same comment as above - `opts` was in there so there is a consistent interface for users to interact with.